src/index.js

```diff
@@ -1,14 +1,15 @@
+/* global document */
```

Comment:
Plase set environement as browser instead: https://stackoverflow.com/questions/41858052/solving-linter-error-no-undef-for-document